.github/workflows/daily.yml

```
@@ -1126,11 +1126,11 @@ jobs:
         if: true && !contains(github.event.inputs.skiptests, 'cluster')
         run: ./runtest-cluster ${{github.event.inputs.cluster_test_args}}
 
-  build-macos:
+  build-old-version-macos:
```

```suggestion
  build-old-macos-versions:
```
Is probably more straight forward.